mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -54,6 +85,7 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 	METHOD method_16079 getLootContextBuilder (ZLnet/minecraft/class_1282;)Lnet/minecraft/class_47$class_48;
 		ARG 1 killedByPlayer
 	METHOD method_16080 drop (Lnet/minecraft/class_1282;)V
+	METHOD method_16212 getBlockStateUnderFeet ()Lnet/minecraft/class_2680;
```

Comment:
"Feet" doesn't make sense for all entities, since fish don't have feet, for example. Also, this returns the block *at* the feet of a player, not under them.

It should just be `getBlockState`.